src/metabase/models/collection/graph.clj

```diff
@@ -67,6 +67,7 @@
         honeysql-form           {:select [[:id :id]]
                                  :from   [:collection]
                                  :where  (into [:and
+                                                [:or [:= :type nil] [:not= :type "trash"]]
```

Comment:
Because `NULL` is terrible 😭 we want to include the cases where `type` is `NULL`, and:

```
metabase_20240313112008=# select 1 where null != 'trash';
 ?column? 
----------
(0 rows)

metabase_20240313112008=# select 1 where null is null or null != 'trash';
 ?column? 
----------
        1
(1 row)
```

The answer to `NULL != "trash"` is itself `NULL`, which is falsey... I should probably add a comment here though, because it does look super confusing and unnecessary.

Reply:
oof